### PR TITLE
build: upgrade jest

### PR DESCRIPTION
**Pull request by PavelKirvelTextkernel**
Task [link](https://textkernel.atlassian.net/browse/ONEUI-415)

### CODE REVIEW

**Comment by PavelKirvelTextkernel** [comment]
image

File: package.json
Comment (antipin) [inline package.json:1]: Let's also update the following packages to the latest versions:
* `@testing-library/jest-dom` `5.16.5 -> 5.17.0`
* `jest-environment-jsdom` `29.1.1 -> 29.6.1`

File: package.json [hunk 1]
```diff
@@ -130,7 +130,9 @@
         "eslint-plugin-storybook": "0.6.2",
         "file-loader": "6.2.0",
         "husky": "8.0.3",
-        "jest": "26.6.3",
+        "jest": "29.6.1",
+        "jest-cli": "^29.6.1",
```
Comment (antipin) [inline package.json:134]: Don't use `^`, it's better to specify exact versions

**Review by antipin — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (PavelKirvelTextkernel) [reply package.json:134]: :+1:

File: package.json
Comment (PavelKirvelTextkernel) [reply package.json:1]: :+1:

**Comment by PavelKirvelTextkernel** [comment]
@antipin I've pushed new changes could you please check it ?

**Review by antipin — APPROVED** [review]
@PavelKirvelTextkernel  looks good 👍

**Pull request merged**